src/expandableCalendar/agendaList.js

```diff
@@ -158,7 +177,10 @@ class AgendaList extends Component {
         onScroll={this.onScroll}
         onMomentumScrollBegin={this.onMomentumScrollBegin}
         onMomentumScrollEnd={this.onMomentumScrollEnd}
-        onScrollToIndexFailed={(info) => { console.warn('onScrollToIndexFailed info: ', info); }}
+        onScrollToIndexFailed={(info) => {
+          console.warn('onScrollToIndexFailed info: ', info);
+        }}
+        {...this.props}
```

Comment:
This is risky and should be located here. Please specify the need to override in the description.

Reply:
If you need 'renderSectionHeader' why no exposing a prop?